{{cookiecutter.project_slug}}/{{cookiecutter.package_name}}/app.py

```diff
@@ -21,6 +22,11 @@ def create_app(config: Configuration):
     async def init(app_, loop):  # pylint: disable=unused-variable
         pass
 
+    @app.listener('before_server_stop')
+    async def wait_before_stopping_server(app, loop):  # pylint: disable=unused-variable
```

Comment:
```suggestion
    async def wait_before_stopping_server(_, __):
```
P4; 보통 요렇게 안쓰는 변수들을 처리합니다.

Reply:
일부로 이렇게 섰어요. (다른 함수도 그 스타일로 해서요.) 그래도 바꿔드릴까요?

Reply:
아 다른함수도 이렇게 되어있군요? 그럼 이대로 유지하죠!

Reply:
(그나저나 RainistJordy가 아니네요 당황)

Reply:
오, 그러네요. git config의 global user.email 다른가봐요~